tools/buildtools/topologyeditors/add_top.xml

```diff
@@ -0,0 +1,129 @@
+<tool id="modify_topology" name="Adding New Topology Information" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@">
+    <description>to a GROMACS topology file</description>
+
+    <macros>
+        <import>macros.xml</import>
+        <token name="@GALAXY_VERSION@">0</token>
+    </macros>
+
+    <expand macro="requirements">
+       <requirement type="package" version="3.8">python</requirement>
```

Comment:
why not 3.9?

Reply:
no particular reason, but I can change it to 3.9

Reply:
I think I was still using 3.8, and just added that as a minimum req.

Reply:
changed to 3.9, and left this only in the macros.xml file, and removed this line from add_top.xml and extract_top.xml.